app/services/finance/business_case_summary.rb

```diff
@@ -7,6 +8,8 @@ def initialize(business_case)
     end
 
     def summarize
+      raise BusinessCaseCalculationError if financials.nil?
```

Comment:
I don't really understand the purpose behind the exception here. It feels like we're using an exception for flow control (i.e., as a GOTO).

We could eliminate the exception (and the `rescue` in the controller) by doing:

```ruby
return [] if financials.nil?
```

... and nothing in the view has to change.